python/pyspark/ml/regression.py

```diff
@@ -2126,6 +2127,181 @@ def __repr__(self):
         return self._call_java("toString")
 
 
+@inherit_doc
+class FactorizationMachines(JavaPredictor, HasMaxIter, HasStepSize, HasTol, HasSolver, HasLoss,
+                            JavaMLWritable, JavaMLReadable):
+    """
+    Factorization Machines.
+
+    loss Supports:
+
+    * logisticLoss (default)
```

Comment:
Dumb question but does this render correctly in python docs? I'm not sure this is how bulleted lists work, but am not sure

Reply:
> Dumb question but does this render correctly in python docs? I'm not sure this is how bulleted lists work, but am not sure

I refer to the LinearRegression docs (regression.py: line51). In my experience, python __doc__ just a string, it won't render as a list. But I have not idea, whether spark will use __doc__ to generate docs html.